app/src/main/java/br/com/dionataferraz/vendas/data/local/LocalDataSource.kt

```diff
@@ -0,0 +1,51 @@
+package br.com.dionataferraz.vendas.data.local
+
+import android.util.Log
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+class LocalDataSource {
+
+    private val database: VendasDatabase by lazy {
+        VendasDatabase.getInstance()
+    }
+
+    suspend fun inserir(accountBalance: Double, type: Type, date: String) {
+        return withContext(Dispatchers.IO) {
+            database.DAO()
+                .insertAccount(
+                    AccountEntity(
+                        accountBalance = accountBalance,
+                        type = type,
+                        date = date
+                    )
+                )
+
+        }
+    }
+
+    suspend fun buscar(): List<AccountModel> {
+        return withContext(Dispatchers.IO) {
+            database.DAO()
+                .getAccount().map {
+                    AccountModel(
+                        accountBalance = it.accountBalance,
+                        type = it.type,
+                        date = it.date
+                    )
+                }
+
+        }
+    }
+
+    suspend fun findBalance(): Double {
+        return withContext(Dispatchers.IO) {
+            val aaa = database.DAO().findBalance()
+            Log.e("dataSource ", aaa.toString())
+            aaa
```

Comment:
Somente o find era necessário, mas deve ter esquecido de remover
```suggestion
            database.DAO().findBalance()
```